minimal-webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const pkg = require('./package.json');
+const path = require('path');
+const webpack = require('webpack');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+
+const production = process.env.NODE_ENV === 'production' || false;
+
+const banner = `clipboard.js v${pkg.version}
+https://clipboardjs.com/
+
+Licensed MIT © Zeno Rocha`;
+
+module.exports = {
+    entry: path.resolve(__dirname, 'src') + '/clipboard-minimal.js',
+    mode: 'production',
+    output: {
+        filename: production ? 'clipboard.min.js' : 'clipboard.js',
+        path: path.resolve(__dirname, './dist-minimal'),
+        library: 'clipboard',
+        globalObject: 'this',
+        libraryExport: 'default',
+        // libraryTarget: 'umd',
+        libraryTarget: 'window',
```

Comment:
What do you think about to remove this unused libraryTarget from code and maintain just the `window`?

Reply:
I can can change that